lib/core/bytes.nit

```diff
@@ -303,6 +318,64 @@ redef class Text
 		end
 		return new FlatString.with_infos(outns, ln * 2, 0, ln * 2 - 1)
 	end
+
+	# Return a `Bytes` instance where Nit escape sequences are transformed.
+	#
+	#     var s = "\\n"
+	#     assert s.length        ==  2
+	#     var u = s.unescape_to_bytes
+	#     assert u.length        ==  1
+	#     assert u[0]            ==  0x0Au8
```

Comment:
instead of ugly length and unicode, cannot you just use `hexdigest` to be more staightforward

    assert u.hexdigest == "0A"
    assert u2.hexdigest == "41???????"